mazda_2017.dbc

```diff
@@ -556,22 +556,20 @@ BO_ 535 CURVE_CTRS: 8 XXX
 
 BO_ 540 CRZ_CTRL: 8 XXX
  SG_ NEW_SIGNAL_6 : 10|1@0+ (1,0) [0|1] "" XXX
- SG_ NEW_SIGNAL_9 : 31|1@0+ (1,0) [0|255] "" XXX
- SG_ ACC_GAS_MAYBE2 : 29|1@0+ (1,0) [0|1] "" XXX
  SG_ HANDS_OFF_STEERING : 48|1@0+ (1,0) [0|1] "" XXX
  SG_ HANDS_ON_STEER_WARN : 59|4@0+ (1,0) [0|255] "" XXX
  SG_ CRZ_ACTIVE : 3|1@0+ (1,0) [0|1] "" XXX
  SG_ CRZ_AVAILABLE : 17|1@0+ (1,0) [0|255] "" XXX
  SG_ DISTANCE_SETTING : 20|3@0+ (1,0) [0|7] "" XXX
  SG_ MSG_1_INV : 1|1@0+ (1,0) [0|1] "" XXX
  SG_ MSG_1_COPY : 9|1@0+ (1,0) [0|1] "" XXX
- SG_ ACC_GAS_MAYBE : 23|1@0+ (1,0) [0|31] "" XXX
  SG_ ACC_ACTIVE_2 : 52|1@0+ (1,0) [0|1] "" XXX
- SG_ NEW_SIGNAL_10 : 30|1@0+ (1,0) [0|1] "" XXX
  SG_ MSG_1 : 0|1@0+ (1,0) [0|3] "" XXX
  SG_ 5_SEC_DISABLE_TIMER : 45|3@0+ (1,0) [0|7] "" XXX
  SG_ NEW_SIGNAL_3 : 13|1@0+ (1,0) [0|3] "" XXX
  SG_ MSG_1_INV_COPY : 8|1@0+ (1,0) [0|7] "" XXX
+ SG_ RADAR_HAS_VEHICLE_AHEAD : 23|1@0+ (1,0) [0|1] "" XXX
+ SG_ RADAR_VEHICLE_AHEAD_RELATIVE_DISTANCE : 31|3@0+ (1,0) [0|5] "" XXX
```

Comment:
these are typically referred to as "leads". can you replace all "vehicle aheads" with "lead"?

Reply:
Sure, I'll update it. I am using the words 'vehicle ahead' only because that is what the manual said.

Reply:
Addresses the issue.